### DISCUSSION TITLE
Add multiple tasks and their respective subtasks easily with Markdown

### DISCUSSION THREAD

**Discussion by chenyenru** [super-productivity/super-productivity#2606] [Ideas]
Hi, I'm amazed by Super Productivity's functionalities. It's so convenient, and the time-tracking function was like a godsend for my ADHD brain.

And in the hope of improving this fantastic software, I'd like to recommend adding a feature that auto-converts markdown checklists to tasks and subtasks and auto-interprets the time written in some tasks as each respective task's schedule.

It would be very convenient to have this kind of task view. Thank you!  
```
- [ ] Correct P1 Practice Mon 19:30
	- [ ] 1
	- [ ] 2
- [ ] Vectors
	- [ ] Shortest Distance
	- [ ] 3 Planes Intersection
- [ ] Counting Principles
- [ ] Maclaurin Series
- [ ] Prob Distribution
```

**Comment by johannesjo** [comment]
Thanks for the suggestion. I think there might be already on open issue somewhere asking for the same thing :)

**Reply by chenyenru** [reply]
Gotcha, thanks!

**Comment by mathisgauthey** [comment]
Hey there, I organise my stuff on Obsidian and do my daily tasks on SupProd. I would then benefit from some batch addition like Todoist does, creating as many tasks as lines and replacing markdown tasks `- [ ]` in the making. Is there any open similar request ? I couldn't find something else than this one.

**Comment by TheGallant** [comment]
Any updates on this? ;-;
